tests/bugfixes/github/test_issue_1503.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+import unittest
+
+# test needs system_tests.BT.vv['enable_bmff']=1
+bSkip=system_tests.BT.verbose_version().get('enable_bmff')!='1'
+if bSkip:
+    raise unittest.SkipTest('*** requires enable_bmff=1 ***')
+
+class issue_1503_jxl_bmff(metaclass=system_tests.CaseMeta):
```

Comment:
It looks like you also need to add here:

```python
@unittest.skip("Skipping test using option -pR (only for Debug mode)")
```
Right now the tests is failing on travis in the Release mode builds.

Reply:
Yes, that's one fix.  I used the option -pS which is enabled on release builds.  For bmff files, -pS and -pR are identical.  I did that to support test_pr1475.py.